### PR TITLE
Add compose plugin in pdc client

### PR DESCRIPTION
**Pull request by simozhan**
JIRA: PDC-1022

### CODE REVIEW

**Comment by bliuredhat** [comment]
look good for me

**Comment by erichuanggit** [comment]
Since tests for new client infrastructure has been merged, I would suggest adding unit test. But it is all right separated in another patch.

**Comment by simozhan** [comment]
@erichuanggit, @lubomir Thanks for the review. Tests added and issue fixed.

**Comment by lubomir** [comment]
Looks good. Please don't use RHEL data in tests, though.

**Comment by simozhan** [comment]
Yes. it's my bad.thanks for pointing this out. fixed

File: pdc_client/tests/compose/tests.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        self.runner.setup()
+        self.compose_detail = {
+            "compose_id": "RHEL-7.0-20130203.7",
+            "compose_date": "2013-02-03",
+            "compose_type": "nightly",
+            "compose_respin": 0,
+            "release": "aifv-5802",
+            "compose_label": "RC-3.0",
+            "deleted": False,
+            "rpm_mapping_template": "https://pdc-dt.app.qa.eng.nay.redhat.com/rest_api/v1/composes/RHEL-7.0-20130203.7/rpm-mapping/{{package}}/",
```
Comment (erichuanggit) [inline pdc_client/tests/compose/tests.py]: please update qa dt server to example url

Comment (simozhan) [inline pdc_client/tests/compose/tests.py]: Yes. Already updated.

**Pull request closed without merging**